packages/cashier/src/types/account.types.ts

```diff
@@ -2,24 +2,29 @@
 /* ACCOUNT TYPES                                                              */
 /* -------------------------------------------------------------------------- */
 import { DetailsOfEachMT5Loginid, TransferBetweenAccountsResponse } from '@deriv/api-types';
-import { PlatformIcons } from '../assets/svgs/trading-platform/index';
+
+export type TPlatformIcon = 'Derived' | 'Financial' | 'Options' | 'CFDs';
 
 export type TAccount = {
     balance?: string | number;
     currency?: string;
     disabled?: boolean;
     error?: JSX.Element | string;
     is_crypto?: boolean;
-    is_dxtrade?: boolean;
     is_derivez?: boolean;
+    is_dxtrade?: boolean;
     is_mt?: boolean;
     market_type?: string;
     nativepicker_text?: string;
-    platform_icon?: string & keyof typeof PlatformIcons;
+    platform_icon?: TPlatformIcon;
+    status?: string;
     text?: JSX.Element | string;
     value?: string;
 };
 
+// This error will be removed when api-types get updated and then we can will import the correct type
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
 export type TTransferAccount = TransferBetweenAccountsResponse['accounts'][number] & {
```

Comment:
```suggestion
export type TTransferAccount = NonNullable<TransferBetweenAccountsResponse['accounts']>[number] & {
```

Reply:
It will produce TS errors in other components, as some properties are `null` and the `null` value has been used in some cases. @farzin-deriv